Source/WTF/wtf/StdLibExtras.h

```diff
@@ -155,6 +155,36 @@ inline bool is8ByteAligned(void* p)
     return !((uintptr_t)(p) & (sizeof(double) - 1));
 }
 
+inline std::byte* alignedBytes(std::byte* pointer, size_t alignment)
```

Comment:
For the most part, @cdumez has convinced me to just use `uint8_t` for generic bytes and not start to use `std::byte`. Maybe you can convince me otherwise, after all `uint8_t` and `LChar` are the same type and one advantage of `std::byte` is that it’s not the same as any other type!

Reply:
Other than some extra overloads, what's the argument?

Reply:
I guess maybe extra type conversions?

Reply:
I would think that would be just a temporary problem while making the move from uint8_t to std::byte for actual opaque byte buffers. 

But either way, this is entirely constrained to the data structure, so if we decide we really don't like std::byte, changing it will be very easy.